pkg/types/baremetal/platform.go

```diff
@@ -33,10 +24,16 @@ type Platform struct {
 	// Default is qemu:///system
 	LibvirtURI string `json:"libvirtURI,omitempty"`
 
-	// IronicURI is the identifier for the Ironic connection.  It must be
-	// reachable from the host where the installer is run.
+	// ClusterProvisioningIP is the IP on the provisioning host private/provisioning
```

Comment:
I don't follow what this field is doing??

`provisioning host` what is this? the host that is getting provisioned to join the cluster as master/compute ?

`ClusterProvisioningIP is the IP on the provisioning host private/provisioning network for the host running the installer` this sentence isn't making a whole lot of sense to me _ can we try to make this a little simpler for non-expert person like me_

Reply:
Ah yeah, the comment is garbled because of some more sed-replacement of a previous name, I'll fix this up.

To clarify, there are two places where the provisioning services (e.g ironic and dependencies) run, first on the bootstrap VM (accessible via BootstrapProvisioningIP to provision the master nodes via terraform), then later on the cluster managed via the Baremetal Operator, at this point the IP used changes to ClusterProvisioning IP.

Note that in both cases the network is a dedicated provisioning network, so unrelated to the normal pod networking and OCP controlplane (it's a dedicated second NIC for PXE boot etc)